src/core/Notification/BaseNotification/BaseNotification.baseStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseNotificationBaseStyles = (theme: SuomifiTheme) => css`
+  &.fi-notification {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    p {
+      font-size: 16px;
```

Comment:
Can we really specify in our components that "if the user adds content of type X, it should have these styles"? I don't think so. What are your thoughts? Discuss before doing anything.

Reply:
Yes, this kind of styling is not wanted. If paragraph is not used by the library user, Notification has broken styles.

Reply:
Fixed, removed p{}